app/api/search/specs/search.spec.js

```diff
@@ -820,6 +820,19 @@ describe('search', () => {
       const { rows } = await search.search(query, 'en');
       expect(rows).toEqual([expect.objectContaining({ title: 'Batman finishes en' })]);
     });
+
+    it('when values is empty should not filter', async () => {
+      const query = {
+        customFilters: {
+          generatedToc: {
+            values: [],
+          },
+        },
+      };
+
+      const { rows } = await search.search(query, 'en');
+      expect(rows.length).not.toBe(0);
```

Comment:
Do you mean change to `expect(rows.length).not.toBe(1);`. When values is true it returns 1.

Reply:
No @Zasa-san, I mean something like `expect(rows.length).toBe(13)`, which I think is the expected number on unfiltered rows, because if you filter for example with `generatedToc: { values: ['true'] }`, rows has one element, what still makes the test pass.